docs/src/main/asciidoc/ap4k.adoc

```diff
@@ -1,20 +1,18 @@
-= {project-name} - Deploying an Application to Kubernetes and OpenShift
+= {project-name} - Generating Kubernetes resources
 
 This guide covers:
 
-* The deployment of the application to Kubernetes
-* The deployment of the application to OpenShift (TODO)
+* Generating Kubernetes resources based on sane defaults and user supplied configuration.
```

Comment:
Maybe we could just have a simple sentence instead of a bullet point? That would be more natural.

Reply:
Makes sense looking at that the other guides do - Fixed